app/plugins/Files/js/loaders.js

```diff
@@ -87,6 +112,14 @@ addResultListener('ascii-loaded', function(result) {
 	update();
 });
 
+//Confirm deletion popup
+function confirmDelete(nickname) {
+	eID('confirm-delete').querySelector('.title').innerHTML = 'Confirm to delete';
```

Comment:
You're right: I think the confirmation string could just be defined in HTML.

And yes, I realize there were too many changes in both of my pull requests. I was oblivious to the constraints of a multi-developer effort. Future PRs will be much smaller and more focused :-)